PKHeX.WinForms/Program.cs

```diff
@@ -93,6 +94,12 @@ private static void CurrentDomain_UnhandledException(object sender, UnhandledExc
                 // Todo: make this translatable
                 ErrorWindow.ShowErrorDialog("An unhandled exception has occurred.\nPKHeX must now close.", ex, false);
             }
+            catch (MissingMethodException)
+            {
+                // Todo: make this translatable
+                MessageBox.Show(".NET Framework 4.6 needs to be installed for this version of PKHeX to run.", "PKHeX Error", MessageBoxButtons.OK, MessageBoxIcon.Stop);
+                Process.Start(@"https://www.microsoft.com/en-us/download/details.aspx?id=48130");
```

Comment:
Have you tested the `Process.Start` in an environment that doesn't have .Net 4.6?  Quite frankly, I'm amazed that the code runs enough to get to the generic backup error message, but I wouldn't exactly trust any bit of code to work properly.